CleverTapSDKTests/CleverTapInstanceTests.m

```diff
@@ -70,4 +74,134 @@ - (void)test_clevertap_get_global_instance {
     XCTAssertEqualObjects([[instance config] accountToken], @"test");
 }
 
+- (void)testSetOptOutFailsWhenBothNO {
```

Comment:
This test fails for me. 
Shouldn't `profilePush:` always be called when calling `setOptOut:allowSystemEvents:`?

Reply:
Must have written reject instead of verify. Fixed